packages/sentry/package.json

```diff
@@ -29,7 +35,7 @@
     "test": "jest"
   },
   "dependencies": {
-    "@sentry/browser": "^6.12.0",
-    "@sentry/node": "^6.12.0"
+    "@sentry/browser": "^6.12.0 || ^7.47.0",
+    "@sentry/node": "^6.12.0 || ^7.47.0"
```

Comment:
I think we could just update to the latest? Any reason to keep the old one around?

Reply:
hmm, I thought I removed v6

Reply:
I missed the package.json but had committed the yarn.lock 🤦